app/src/test/kotlin/fr/free/nrw/commons/nearby/NearbyParentFragmentPresenterTest.kt

```diff
@@ -164,60 +175,68 @@ class NearbyParentFragmentPresenterTest {
      * invisible and location is slightly changed
      */
-    @Test @Ignore
+    @Test
     fun testUserNotTrackedWhenCurrentLocationMarkerInvisible() {
         expectMapAndListUpdate()
+        verify(nearbyParentFragmentView).enableFABRecenter()
         whenever(nearbyParentFragmentView.isCurrentLocationMarkerVisible()).thenReturn(false)
+        whenever(nearbyParentFragmentView.lastMapFocus).thenReturn(LatLng(2.0, 1.0, 0.0F))
+        whenever(nearbyParentFragmentView.mapCenter).thenReturn(null);
         nearbyPresenter.updateMapAndList(LocationChangeType.LOCATION_SLIGHTLY_CHANGED)
-        verify(nearbyParentFragmentView).enableFABRecenter()
         verify(nearbyParentFragmentView).isNetworkConnectionEstablished()
-        verify(nearbyParentFragmentView).getLastLocation()
+        verify(nearbyParentFragmentView).getLastMapFocus()
+        verify(nearbyParentFragmentView).getMapCenter()
         verify(nearbyParentFragmentView).isCurrentLocationMarkerVisible()
         verifyNoMoreInteractions(nearbyParentFragmentView)
     }
-
+    // Removed this test function because of change in functionalities due to  Mapbox to osmdroid transition.
     /**
-     * Test search this area button became visible after user moved the camera target to far
+         * Test search this area button became visible after user moved the camera target to far
      * away from current target. Distance between these two point is 111.19 km, so our camera target
      * is at outside of previously searched region if we set latestSearchRadius below 111.19. Thus,
      * setSearchThisAreaButtonVisibility(true) should be verified.
      */
-    @Test @Ignore
-    fun testSearchThisAreaButtonVisibleWhenMoveToFarPosition() {
-        NearbyController.latestSearchLocation = Mockito.spy(LatLng(2.0, 1.0, 0.0F))
-        mapboxCameraTarget = Mockito.spy(com.mapbox.mapboxsdk.geometry.LatLng(1.0, 1.0, 0.0))
-        // Distance between these two point is 111.19 km
-        NearbyController.latestSearchRadius = 111.0 * 1000 // To meter
-        whenever(nearbyParentFragmentView.isNetworkConnectionEstablished()).thenReturn(true)
-        nearbyPresenter.onCameraMove(mapboxCameraTarget)
-        verify(nearbyParentFragmentView).setSearchThisAreaButtonVisibility(true)
-    }
+//    @Test
+//    fun testSearchThisAreaButtonVisibleWhenMoveToFarPosition() {
+//        NearbyController.latestSearchLocation = Mockito.spy(LatLng(2.0, 1.0, 0.0F))
+//        mapboxCameraTarget = Mockito.spy(com.mapbox.mapboxsdk.geometry.LatLng(1.0, 1.0, 0.0))
+//        // Distance between these two point is 111.19 km
+//        NearbyController.latestSearchRadius = 111.19 * 1000 // To meter
+//        whenever(nearbyParentFragmentView.isNetworkConnectionEstablished()).thenReturn(true)
+//        nearbyPresenter.onCameraMove(mapboxCameraTarget)
+//        verify(nearbyParentFragmentView).setSearchThisAreaButtonVisibility(true)
+//    }
```

Comment:
Please just let this test as `@Ignore`, it is better than commented out.

Reply:
Resolved